bridge/slack/helpers_test.go

```diff
@@ -0,0 +1,26 @@
+package bslack
+
+import (
+	"testing"
+)
```

Comment:
If there's only one import our `goimports` linter will want this to be on one line without the parentheses. 🤷‍♂️